modules/gsp-cluster/main.tf

```diff
@@ -50,12 +50,17 @@ module "k8s-cluster" {
   controller_instance_type = "${var.controller_instance_type}"
   worker_instance_type     = "${var.worker_instance_type}"
   s3_user_data_policy_arn  = "${aws_iam_policy.s3-user-data-policy.arn}"
-  nat_gateway_ips          = "${aws_nat_gateway.cluster.*.public_ip}"
+
+  apiserver_allowed_cidrs = ["${concat(
+      list(aws_vpc.network.cidr_block),
```

Comment:
This may be too open....it could be replaced with something like:
```
aws_subnet.cluster-public.*.cidr_block,
aws_subnet.cluster-private*.cidr_block,
```
Practically they're the same; so I went with the simplest form.

Reply:
Entire VPC network range is fine (and expected I would say)